src/columnresizing.js

```diff
@@ -205,7 +205,11 @@ function zeroes(n) {
 function handleDecorations(state, cell) {
   let decorations = []
   let $cell = state.doc.resolve(cell)
-  let table = $cell.node(-1), map = TableMap.get(table), start = $cell.start(-1)
+  let table = $cell.node(-1)
+  if (!table) {
+    return DecorationSet.create(state.doc, decorations)
```

Comment:
couldn't you used `DecorationSet.empty` here instead? https://prosemirror.net/docs/ref/#view.DecorationSet^empty

Reply:
Oh yes! And that would make the intention clearer. Fixed [here](https://github.com/Asana/prosemirror-tables/blob/052da542eede0b8aff82126b5f87e802136a961e/src/columnresizing.js#L210).